### PR TITLE
docs: unfined defineNuxtPlugin migration guide for nuxt bridge #22433

### PR DESCRIPTION
**Pull request by adrienZ**
### 🔗 Linked issue

#22433 

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by wattanx — APPROVED** [review]
Thank you ❤️

File: docs/6.bridge/3.bridge-composition-api.md [hunk 1]
```diff
@@ -94,7 +94,9 @@ Remove the `defineNuxtPlugin` wrapper:
 ```diff
 - import { defineNuxtPlugin } from '@nuxtjs/composition-api'
 - export default defineNuxtPlugin((ctx, inject) => {})
-+ export default (ctx, inject) => {}
++ export default defineNuxtPlugin(nuxtApp => {
```
Comment (wattanx) [inline docs/6.bridge/3.bridge-composition-api.md:97]: It's possible to provide an explanation similar to Nuxt 3.
https://nuxt.com/docs/migration/plugins-and-middleware#plugins

```ts
export default defineNuxtPlugin(nuxtApp => {
  // now available on `nuxtApp.$injected`
  nuxtApp.provide('injected', () => 'my injected function')

  // You can alternatively use this format, which comes with automatic type support
  return {
    provide: {
      injected: () => 'my injected function'
    }
  }
})
```

**Review by wattanx — COMMENTED** [review]
Sorry, A description of the `defineNuxtPlugin` compatible with Nuxt 3 can be found at https://nuxt.com/docs/bridge/plugins-and-middleware.

What is explained here is how to migrate from `@nuxtjs/composition-api` to Nuxt Bridge's `capi.legacy`.

**Review by wattanx — CHANGES_REQUESTED** [review]
I have approve it by mistake.(I don't know how to cancel approve. 😢 )

**Comment by danielroe** [comment]
Maybe, instead of making this change to the code example, we could add a note after the example that there is a new `defineNuxtPlugin` and link to the page where it is explained?

**Comment by wattanx** [comment]
> Maybe, instead of making this change to the code example, we could add a note after the example that there is a new defineNuxtPlugin and link to the page where it is explained?

I think that suggestion is a good one.

**Review by wattanx — APPROVED** [review]
looks good to me!

**Review by danielroe — APPROVED** [review]

**Pull request merged**